ports/pc/main.c

```diff
@@ -28,6 +28,30 @@ const uint16_t synth_freq = 32000;
 struct voice_ch_t poly_voice[16];
 struct poly_synth_t synth;
 
+/* Read a script instead of command-line tokens */
+static int readScript(const char* name, int* argc, char*** argv) {
```

Comment:
I've got a preference that function names and identifiers use "snake_case" rather than "camelCase", but that's an easy fix to make at this end. :-)

Reply:
You are right, thanks for fixing this for me...